examples/lib/defaultHtmlStepUi.js

```diff
@@ -331,7 +331,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
               .data('initValue', step.options[i]);
           if (inputs[i].type.toLowerCase() === 'select')
             $step('div[name="' + i + '"] select')
-              .val(step.options[i])
+              .val(String(step.options[i]))
```

Comment:
Just make sure that this does not break any other module.

Reply:
No, it does not because everything is a string in the DOM.

Reply:
Ok great!